packages/squidex/schema/schemas/events.json

```diff
@@ -342,7 +342,7 @@
         ]
       },
       {
-        "name": "Hidden",
+        "name": "hidden",
```

Comment:
the 'Hidden' property never should have got into master however it did as part of:
https://github.com/yldio/asap-hub/commit/13c01e96218e013a0dbd9939bfca47f386fa8f17